### ISSUE TITLE
How could I know which pretrained model is best?

### ISSUE THREAD

**Issue by fbotp** [jzhang38/LPD#2]
I reproduced your code and noticed that during the pretraining stage it will save the model at each `save_step`, but how do you know which one is the best? I tried adding a parameter called `total_loss` to save the model with the smallest loss value. However, in 20 epochs, I found that the `total_loss` kept decreasing, and in the finetune phase, the model with the smallest loss did not perform as described in the paper, so where did I go wrong? Waiting for your reply. Thanks
